CHANGELOG.md

```diff
@@ -60,6 +60,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * [#7622](https://github.com/osmosis-labs/osmosis/pull/7622) Create/remove tick events.
 * [#7623](https://github.com/osmosis-labs/osmosis/pull/7623) Add query for querying all before send hooks
 * [#7622](https://github.com/osmosis-labs/osmosis/pull/7622) Remove duplicate CL accumulator update logic.
+* [#7503](https://github.com/osmosis-labs/osmosis/pull/7677) feat(x/incentives): Restrict lockable durations to 2 weeks.
+* [#7625](https://github.com/osmosis-labs/osmosis/pull/7625) Remove duplicate CL accumulator update logic.
```

Comment:
Did this get in accidentally?

Reply:
yes, has been deleted in the most recent commit